tests/ci/commit_status_helper.py

```diff
@@ -84,7 +84,7 @@ def get_commit(gh: Github, commit_sha: str, retry_count: int = RETRY) -> Commit:
 
 def post_commit_status(
     commit: Commit,
-    state: StatusType,
+    state: Union[StatusType, str],
```

Comment:
Oh no, here we're again..

Anything but `StatusType` breaks the `commit.create_status(`. Please, don't ease this part. It must be fixed everywhere at execution